src/driver_humidity.cpp

```diff
@@ -16,27 +16,77 @@
  */
 
 #include "./driver_humidity.h"
-
 #include "./src/driver.pb.h"
-#include "matrix_hal/humidity_data.h"
+#include "matrix_hal/humidity_data.h" 
```

Comment:
You need to use clang-format to format the files. That way there is no need to think of indentation issues.